.eslintrc.json

```diff
@@ -1,6 +1,10 @@
 {
     "extends": ["plugin:jsdoc/recommended", "./configs/eslint-common.json"],
     "rules": {
+        "react/jsx-props-no-spreading": "off",
+        "no-void": "off",
+        "react/prop-types": "off",
```

Comment:
We want to disable it only for .tsx files. I think it can be achieved by similar code to this in file: cloudify-ui-components/.eslintrc.json

Reply:
If it's to be done globally then `config/eslint-common-react.json` would be the place.

Reply:
I addressed it in febf9f1 and in c4433ef.